app/views/Notification.js

```diff
@@ -106,20 +114,26 @@ class NotificationScreen extends Component {
   };
 
   colorfill(data) {
-    var color = 'green';
+    var color = colors.BROWN;
```

Comment:
Yeah, I considered doing that.  Was being lazy since it was already in the colors structure as colors.RED, etc.  But will do -- that makes more sense to me, too.